ci/steps/brew-bump.sh

```diff
@@ -13,27 +28,65 @@ main() {
   echo "Cloning cdrci/homebrew-core"
   git clone https://github.com/cdrci/homebrew-core.git
 
+  # Make sure the git clone step is successful
+  if [[ $(directory_exists "homebrew-core") -eq 1 ]]; then
+    echo "git clone failed. Cannot find homebrew-core directory."
+    ls -la
+    exit 1
+  fi
+
   echo "Changing into homebrew-core directory"
   cd homebrew-core && pwd
```

Comment:
Unrelated to your change but it looks like using pushd/popd would be better here, since they print the path stack when they change things. On line 39 we're changing into homebrew-core and then on line 51 we're going back up a level

Reply:
I haven't used pushd/popd but Ill take your word and see if that's an easy refactor!

Reply:
I think the proposed change would be, on line 39, change `cd homebrew-core` to `pushd homebrew-core` - this remembers the previous location and enters a directory. Then on line 51, call `popd` which changes to the directory you were in before you called pushd - it's a stack.

Reply:
Woah! That's so cool. Thanks for the guidance there. Very very easy to make the change. TIL!